app/Actions/Photos/DeletePhotoAction.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+
+namespace App\Actions\Photos;
+
+
+use App\Models\Photo;
+use Illuminate\Support\Facades\File;
+use Illuminate\Support\Facades\Storage;
+
+class DeletePhotoAction
+{
+    public function run(Photo $photo)
+    {
+        if (app()->environment('production')) {
```

Comment:
I'd actually argue for removing most of the try-catch blocks on the code 😅. It normally shouldn't fail, and if it does, that means we have done something wrong somewhere else, and we'd want the error to bubble up. In that case, it would be easier for us to debug if we had the whole stack trace of the error instead of just the message.

I'd recommend we use try-catch blocks to do extra actions or steps in case something fails, instead of just logging it.